layers/+lang/python/README.org

```diff
@@ -94,7 +94,7 @@ opening a python buffer:
 To fix this, install the =anaconda-mode= [[https://github.com/proofit404/anaconda-mode/wiki][anaconda-deps]] by hand:
 
 #+BEGIN_SRC sh
-  pip install --upgrade "jedi>=0.9.0" "json-rpc>=1.8.1" "service_factory>=0.1.5"
+  pip install --upgrade "jedi>=0.9.0,<0.11.0" "json-rpc>=1.8.1" "service_factory>=0.1.5"
```

Comment:
I actually see that `anaconda-mode` requires `jedi >= 0.13.0`. [Source](https://github.com/proofit404/anaconda-mode/commit/150390968bb0c5180026932d27255483d690e00f). Am I missing something?

Reply:
Could be because locally my anaconda-mode is on 0.1.6 and there they talked about this problem https://github.com/proofit404/anaconda-mode/tree/v0.1.6.
Checking M-x package-list-packages it shows this version `anaconda-mode      20161121.1137 ` installed. I think I misunderstood and thought that spacemacs would update them every time I ran M-m f e R

Reply:
No. Please try upgrading your packages and try again 😸 If it works, I would actually update your PR with a new version of `jedi`.

Reply:
I did this now `cd ~/.emacs.d && git pull`, installed jedi with `pip install -U jedi` which installed `jedi-0.13.1`, ran `M-x configuration-layer/update-packages` and it seems to work normally.

Reply:
Thank you anyway!